packages/UI/src/components/atoms/Radio/StyledRadio.ts

```diff
@@ -16,13 +16,15 @@ export const Option = styled.label`
   p {
     pointer-events: none;
   }
+
+  margin-bottom: 0.25rem;
 `;
 
 export const Input = styled.input`
   height: 24px;
-  width: 24px;
+  /* width: 24px; */
```

Comment:
Why is this commented out?

Reply:
hard-coded pixel width was causing the radio 'circles' to be 'ovals' that were stretched out